foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -112,16 +125,15 @@ Edge.sendEvent(experienceEvent: experienceEvent) { (handles: [EdgeEventHandle])
 
 ### Objective-C
 
-#### Syntax
+**Syntax**
 
-```text
-// Swift Edge
-static func sendEvent(experienceEvent: ExperienceEvent, _ completion: (([EdgeEventHandle]) -> Void)? = nil)
+```objectivec
++ (void) sendExperienceEvent:(AEPExperienceEvent * _Nonnull)> completion:^(NSArray<AEPEdgeEventHandle *> * _Nonnull)completion>
```

Comment:
should the completion handler be nullable here?

Reply:
This is what I saw when get the API in xcode
![sendExperienceEvent](https://user-images.githubusercontent.com/29806547/150194465-05f7878f-9bc3-465d-9dc2-a4ec09281322.png)

It is set to Nonnull for completion